src/styles/variables.scss

```diff
@@ -0,0 +1,4 @@
+$base-color: #060b35;

```

Comment:
This file better name with '_'  in beginning and move to utils block. And all blocks remove to `blocks` folder.

More about file structure in scss in this [video, timing 1:00](https://mate.academy/learn/css-advanced/sass#/video/931)